src/Ml.Cli.WebApp/ClientApp/src/Server/Project/Page/Page.js

```diff
@@ -5,9 +5,18 @@ import Users from './Users';
 import Title from '../../../TitleBar';
 import ActionBar from './ActionBar';
 import './Page.scss';
+import Lock from "../../shared/Lock/Lock";
+import ConfirmModal from "../../shared/ConfirmModal/ConfirmModal";
 
-const Page = ({ project, dataset, users, group, annotationsStatus, onExport, user }) => (
+const Page = ({ project, dataset, users, group, annotationsStatus, isModalOpened, onExport, user, lock }) => (
```

Comment:
**suggestion**: make an unit test

Reply:
It's already tested: the Page.container component renders the Page component with resilience and needed methods. There are several unit tests on Page.container component